iphone/Classes/TiUIApplicationShortcutsProxy.m

```diff
@@ -211,6 +212,14 @@ - (UIApplicationShortcutIcon *)findIcon:(id)value
     return [UIApplicationShortcutIcon iconWithTemplateImageName:[self urlInAssetCatalog:value]];
   }
 
+#ifdef IS_SDK_IOS_13
+  if ([value isKindOfClass:[TiBlob class]] && [TiUtils isIOSVersionOrGreater:@"13.0"]) {
+    TiBlob *blob = (TiBlob *)value;
+    if (blob.type == TiBlobTypeSystemImage) {
+      return [UIApplicationShortcutIcon iconWithSystemImageName:blob.systemImageName];
```

Comment:
A wild guess here: Instead of creating new blob types and all this handling, why not just introduce a new `systemIcon` String proxy property and pass it directly to the `iconWithSystemImageName` constructor? That'd probably be around 10 lines of code in total.

Reply:
@hansemannn 1. UIApplicationShortcutIcon has 4 type of icons, which are getting handled from same place in SDK. And developer have to pass proper value to 'icon' property and it gets internally handled on basis of type.
2. Only one type of icon can be set at a time. Having 2 property will create confusion.
3. Similar type of handling sf symbols will be better for developer.